test/Microsoft.Data.Analysis.Interactive.Tests/Microsoft.Data.Analysis.Interactive.Tests.csproj

```diff
@@ -1,15 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <PropertyGroup>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
-        <NoWarn>$(NoWarn);MSML_PrivateFieldName;MSML_ExtendBaseTestClass</NoWarn>
-    </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <NoWarn>$(NoWarn);MSML_ExtendBaseTestClass</NoWarn>
+  </PropertyGroup>
 
-    <ItemGroup>
-        <ProjectReference Include="..\..\src\Microsoft.Data.Analysis.Interactive\Microsoft.Data.Analysis.Interactive.csproj" />
-    </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Microsoft.Data.Analysis.Interactive\Microsoft.Data.Analysis.Interactive.csproj" />
 
-    <!-- register for test discovery in Visual Studio -->
-    <ItemGroup>
-        <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
-    </ItemGroup>
+    <!-- work around https://github.com/dotnet/runtime/issues/45560 until System.Text.Json is fixed and Microsoft.DotNet.Interactive updates to the new version -->
```

Comment:
This looks fine to me. 

I also just discovered that there's both https://www.nuget.org/packages/Microsoft.DotNet.Interactive/1.0.0-beta.21155.3 and https://www.nuget.org/packages/Microsoft.dotnet-interactive/

Reply:
Microsoft.dotnet-interactive is the application / global tool. Microsoft.DotNet.Interactive is the API / library.